internal/repository/verify.go

```diff
@@ -22,47 +22,61 @@ import (
 // another is to create a new RSL entry for the current state.
 var ErrRefStateDoesNotMatchRSL = errors.New("Git reference's current state does not match latest RSL entry") //nolint:stylecheck
 
-func (r *Repository) VerifyRef(ctx context.Context, target string, latestOnly bool) error {
+func (r *Repository) VerifyRef(ctx context.Context, localRefName, remoteRefName string, latestOnly bool) error {
```

Comment:
I generally consider having funcs where you need to pass in zero values as an antipattern, since it's not clear whether these values are actually required or not.

I would recommend moving to variadic options for these if they are not actually required.

Reply:
I wonder if that can lead to confusion in the right order to pass in the strings? We'd also have to do length check for the variadic slice. Would a refToBeVerified opts struct make sense instead? Should we consider moving to opts structs more generally?

Reply:
Have an example where this might be confusing?

typically you'd have an internal opts struct that you populate based on the variadic options. e.g. https://go.dev/play/p/YvJLymVzEmX

Reply:
Got it, I misunderstood your original comment with variadic "options"

Reply:
done!